README.rst

```diff
@@ -15,6 +15,9 @@ Resonance: Learning Mechanical Vibration Engineering Through Computation
 .. image:: https://travis-ci.org/moorepants/resonance.svg?branch=master
    :target: https://travis-ci.org/moorepants/resonance
 
+.. image:: https://mybinder.org/badge.svg
+   :target: https://mybinder.org/v2/gh/moorepants/resonance/master
```

Comment:
May want to change this to link directly to `index.ipynb` once it's usable on Binder.